buildenv

```diff
@@ -29,7 +30,15 @@ zopen_check_results()
 
 zopen_append_to_env()
 {
-  # echo envars outside of PATH, MANPATH, LIBPATH
+
+cat <<ZZ
+# Set LE character set for compilation to ASCII.
+# Set build target to z/os v2r4 untill the build issue on v2r5 and above is fixed.
+export ZOPEN_EXTRA_CFLAGS="-fzos-le-char-mode=ascii -std=gnu11 -mnocsect -fno-short-enums -Wno-error -mzos-target=zosv2r4 -I../src"
```

Comment:
Can you explain how this is used? Typically, if it's a library, it follows the following format:
```
if [ ! -z "\$ZOPEN_IN_ZOPEN_BUILD" ]; then
  export ZOPEN_EXTRA_CFLAGS="\${ZOPEN_EXTRA_CFLAGS} -I\$PWD/include"
  export ZOPEN_EXTRA_CXXFLAGS="\${ZOPEN_EXTRA_CXXFLAGS} -I\$PWD/include"
  export ZOPEN_EXTRA_LDFLAGS="\${ZOPEN_EXTRA_LDFLAGS} -L\$PWD/lib"
  export ZOPEN_EXTRA_LIBS="\${ZOPEN_EXTRA_LIBS} -lssl -lcrypto"
 fi
```
as in the case of openssl

Reply:
I have pushed the changes, can you please review